.github/workflows/test-view.yml

```
@@ -1,51 +1,37 @@
-    name: Test View
+  checks:
+    name: ${{ matrix.action }}
     runs-on: ubuntu-latest
+    strategy:
+        matrix:
+            action:
+                - type-check
+                - lint
+                - test
     steps:
       - uses: actions/checkout@v3
```

actionlint (1.7.4)

the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)